wagtail/contrib/sitemaps/sitemap_generator.py

```diff
@@ -24,14 +24,16 @@ def get_wagtail_site(self):
         return site
 
     def items(self):
-        return (
+        return list(filter(
+            lambda item: item.url is not None,
```

Comment:
What's the situation that necessitates this check on a page by page basis? Or, more generally, isn't it the case that, since this code is iterating over all pages within a site, all pages either will or won't be routable, depending on if `wagtail_serve` is defined?

Reply:
You're right, I'll undo this and take a deeper look to make sure only routable URLs are being listed here. Sorry.